plugin/firewall/policy/expression.go

```diff
@@ -0,0 +1,120 @@
+package policy
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/coredns/coredns/plugin/pkg/rqdata"
+
```

Comment:
fmt

Reply:
ok. Look like my tool is not set the same way as you wish the import to be formatted. 
But the validation in Makefile is ok ...